shared/src/main/scala/mouse/anyf.scala

```diff
@@ -8,4 +12,11 @@ trait AnyFSyntax {
 final class AnyFOps[F[_], A](private val fa: F[A]) extends AnyVal {
   @inline def ||>[G[_]](f: F ~> G): G[A] = f(fa)
   @inline def thrushK[G[_]](f: F ~> G): G[A] = f(fa)
+
+  def liftEitherT[E](implicit F: Functor[F]): EitherT[F, E, A] =
```

Comment:
The function name is the same as in `FOptionOps`. Compiler prefers better fit implicit (`F[Option[A]]` over `F[A] where A = Option[A]`), so it works, however if you feel that those functions could be named differently just give me a shout.

Reply:
This method could be replaced with `fa.thrushK(EitherT.liftK)` where `thrushK` is from `AnyFOps`.

Reply:
Hi @matwojcik 

I guess you mean [FEitherOps](https://github.com/typelevel/mouse/blob/main/shared/src/main/scala/mouse/feither.scala#L91)?

I would prefer a different name, simply because they differ slightly in meaning. The previous method can lift either left or right, whereas this new one lifts right only. 

Do any of`rightEitherT`, `liftRightT`, `liftEitherTR` appeal to you? Or happy to consider another suggestion or pointer to existing precedent in scala...

Reply:
@benhutchison yes, `FEithherOps`, but fot `Option` we have the same issue with name conflict.

When it comes to `liftEitherT` i like `liftRightT` most.

And what's your view on `liftOptionT`?

Reply:
Right.. its parallel isn't it. Some/None vs Right/Left. In same way, the existing method in `foption` lifts an option to either Some / None, whereas the new proposal will lift to Some only.

If we choose `liftRightT` then we choose `liftSomeT` for consistency?

Reply:
Cool, adjusted

Reply:
In `FOptionSyntax` we still have `liftOptionT`. I would find it confusing if for some cases I had to use `liftOptionT` while for some other `liftSomeT` to achieve the same thing.

I'd expect `liftOptionT` to always work on `Some` underneath, the same way as `Option.apply` always returns `Some`.
As for `liftEitherT`, the main use case I've seen for it is in for expressions, where I've got a pure value and only want it wrapped for the types to match.

My personal preference is to stick to `liftEitherT` and `liftOptionT`.

Reply:
Thanks for your input @majk-p 

What Im hearing is you prefer if `liftOptionT` and `liftEitherT` are a bit overloaded and context-sensitive; the name expresses intent to end up with a stack of `OptionT` and `F`, and it should just "do the right thing" for the shape of the type within the `F`.

Whereas Id previously emphasized explicitness for different operations.. but maybe users want "smart" over "explicit"..

Reply:
Thank you @benhutchison, you got my intentions right! I'd vote for smart over explicit here

Reply:
@matwojcik Apologies for change of plans, can we please revert to your original naming, `liftOptionT` & `liftEitherT`

Reply:
@benhutchison Done :)